src/Geopilot.Frontend/public/locale/de/common.json

```
@@ -0,0 +1,57 @@
+{
+  "about": "Über",
+  "appSubTitle": "Online Validierung & Lieferung von Geodaten",
+  "bugTracking": "Bugtracking",
+  "cancel": "Abbrechen",
+  "close": "Schliessen",
+  "codeLicenseInfo": "Der Code steht unter der <licenseLink>MIT Lizenz</licenseLink> im <repositoryLink>Github Repository</repositoryLink> zur Verfügung. Falls Ihnen Bugs begegnen, können Sie dort ein <issuesLink>Issue</issuesLink> eröffnen.",
+  "comment": "Kommentar",
+  "createDelivery": "Lieferung erstellen",
+  "createDeliverySuccess": "Lieferung erfolgreich",
+  "createDeliveryFailed": "Bei der Abgabe ist ein unbekannter Fehler aufgetreten, bitte versuchen Sie es mit einer neuen Validierung.",
```

REQ: Lieferung durchgehend verwenden.
```suggestion
  "createDeliveryFailed": "Bei der Lieferung ist ein unbekannter Fehler aufgetreten, bitte versuchen Sie es mit einer neuen Validierung.",
```